### PR TITLE
Msrk bangalore

### PR DESCRIPTION
**Pull request by msrk-adl**
Adding my pin

### CODE REVIEW

**Comment by sfiggis** [comment]
This will work if you change the format of the file to:

---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---

(making sure to include the --- on top and bottom).

You also need to add "@githubteacher" and "this closes #[add the number for your original issue you made here]". Hope that helps!

**Comment by shajanjp** [comment]
@msrk-adl you are almost there, change `githubHandle`, to `githubhandle`and you are good to go.

File: _pins/msrk.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle: msrk
```
Comment (hollenberry) [inline _pins/msrk.json]: Almost! Change this to `githubHandle` and it should pass.

File: pins/msrk.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (hollenberry) [inline pins/msrk.json]: Also, make sure to delete this file. It's unnecessary.

**Comment by hollenberry** [comment]
@msrk-adl   I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/5260/commits/f51fea0be8676096c4c2b8e3c5549ef5bb1abd25. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green Merge pull request button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**